public/assets/inst.js

```diff
@@ -516,6 +518,7 @@ function runInstruction(raw) { //, RISCV) {
         case 0x37:
             RISCV.gen_reg[((raw >>> 7) & 0x1F)] = signExtLT32_64(((raw & 0xFFFFF000)));
             RISCV.pc += 4;
+            RISCV.instruction_amounts['arithmetic'] = RISCV.instruction_amounts['arithmetic'] + 1;
```

Comment:
what if you define the keys as constants, so you can use the constants to index into your object? this would make it potentially less error prone
```
const keyMath = 'arithmetic'; // variable name up to you
RISCV.instructionAmounts[keyMath]++; 

```

Reply:
done